README.md

```diff
@@ -436,11 +432,7 @@ en-au-ocker:
 
 Contributing
 ------------
-See [CONTRIBUTING.md](https://github.com/stympy/faker/blob/master/CONTRIBUTING.md).
-
-Contact
--------
-Comments and feedback are welcome. Send an email to Benjamin Curtis via the [google group](http://groups.google.com/group/ruby-faker).
+See [CONTRIBUTING.md](https://github.com/factory-helper/factory-helper/blob/master/CONTRIBUTING.md).
```

Comment:
A single email account, if we set one up, could be useful to list in the 'Contact' section